pkg/api/addons.go

```diff
@@ -421,7 +421,7 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 			},
 			{
 				Name:  common.AntreaInstallCNIContainerName,
-				Image: k8sComponents[common.AntreaInstallCNIContainerName],
+				Image: k8sComponents["antrea"+common.AntreaInstallCNIContainerName],
```

Comment:
Both flannel and antrea deliver a "install-cni" container in their respective specs, so this additional key prefix helps to disambiguate between the two in the container image reference hash table.